nifi-commons/nifi-site-to-site-client/src/main/java/org/apache/nifi/remote/client/SiteToSiteClient.java

```diff
@@ -560,7 +560,8 @@ public SiteToSiteClient build() {
         }
 
         /**
-         * @return the configured URL for the remote NiFi instance
+         * @return the configured URL for the remote NiFi instance.
+         * The URL string possibly contains multiple URL entries in comma-separated format.
          */
         public String getUrl() {
```

Comment:
This is concerning to me. This may be a breaking change for some, as it previously returned a String that was a valid URI or URL. It may not be anymore. I would rather see this return only the first URL, and have an additional method added: `List<String> getUrls();`

Thoughts?